scripts/system/edit.js

```
@@ -1369,7 +1369,12 @@ var PropertiesTool = function (opts) {
     });
 
     webView.webEventReceived.connect(function (data) {
-        data = JSON.parse(data);
+        try {
+            data = JSON.parse(data);
+        }
+        catch(e) {
+            return;
```

would it make sense to print something here?  silent failures can be hard to track down